ui/src/app/contexts/error-reporter-context.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import {createContext, useEffect, useState} from 'react';
+
+import {StackdriverErrorReporter} from 'stackdriver-errors-js';
+
+import {configApi} from 'app/services/swagger-fetch-clients';
+import {serverConfigStore as angularServerConfigStore} from 'app/utils/navigation';
```

Comment:
Is there are reason this can't be refactored to use an atom, rather than introducing a new store?

Sidenote: I don't know about characterizing the existing serverConfigStore and "angular" store - in actuality, it was only added to support React code. Presumably the concern is that it is using an RxJS BehaviorSubject (Angular uses RxJS, but the library is independent). But understood that you may have just needed a different name here.

Reply:
That's the intended end state, but this store is used very widely and I didn't want to collapse RW-6351 and RW-6348 into one PR.

Name can be changed to something like `rxjsStore` or `legacyStore` for clarity.